DeepLearningSuite/DeepLearningSuiteLib/FrameworkEvaluator/MassInferencer.h

```diff
@@ -15,7 +15,7 @@ class MassInferencer {
     MassInferencer(DatasetReaderPtr reader, FrameworkInferencerPtr inferencer, const std::string &resultsPath, bool* stopDeployer, double* confidence_threshold = NULL, bool debug=true);
     MassInferencer(DatasetReaderPtr reader, FrameworkInferencerPtr inferencer, double* confidence_threshold = NULL, bool debug=true);
     MassInferencer(DatasetReaderPtr reader, FrameworkInferencerPtr inferencer, bool debug=true);
-    void process(bool writeImages, DatasetReaderPtr readerDetection = NULL);
+    void process(bool writeImages, DatasetReaderPtr readerDetection = NULL, unsigned int *mean_inference_time = 0);
```

Comment:
Since readerDetection is also an optional argument, I cannot pass mean_inference_time without passing readerDetection. A better solution would be to call massInferencer.inferencer->getMeanDurationTime(), this would prevent passing any arguments.